src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts

```diff
@@ -573,6 +573,28 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 	handler: accessor => focusAndSelectHandler(accessor, false)
 });
```

Comment:
This call to `registerCommandAndKeybindingRule` isn't needed anymore. The new action2 also does that

Reply:
I forgot to stage that part. All done.